hack/release.sh

```diff
@@ -45,6 +35,9 @@ website() {
     find website/content/en/${RELEASE_VERSION}/ -type f | xargs perl -i -p -e "s/{{< param \"latest_release_version\" >}}/${RELEASE_VERSION}/g;"
 }
 
-image
+requireCloudProvider
+authenticate
+buildImages $RELEASE_VERSION
+cosignImages
```

Comment:
Do we not sign nightlies?

Reply:
It seems like running cosign will require a human interaction the way it works with stable releases, as of now the snapshots and nightlies do not have this process.

Reply:
it might "just work", maybe worth testing out on a fork: https://github.com/sigstore/cosign/blob/main/KEYLESS.md#identity-tokens

Reply:
I'm gonna note down to experiment with that, It seems to me like the signing can happen without human interaction but then you need to store a key somewhere and that's slightly more complex, but I can do that as a follow up PR if you don't mind.